### ISSUE TITLE
cities.json is outdated

### ISSUE THREAD

**Issue by bennybauer** [eladnava/redalert-android#5]
Looks like `cities.json` is outdated. For example, cities defined under "Golan 2" area actually belong to "Golan 5" area. 
When I tried to reverse engineer Pikud Haoref map (http://www.oref.org.il/1096-he/Pakar.aspx) I saw that Neot Golan, for example, has the same value of Golan 5:
```
{label:"נאות גולן", value:"370", id:"29"}
{label:"גולן 5", value:"370"}
```

while `golan 2` has a different value:
```
{label:"גולן 2", value:"46"}
```

Thanks!

**Comment by eladnava** [comment]
Hi @bennybauer,
Thanks so much for your report. It does appear that the app is using an outdated `cities.json`. This file has been manually extracted using information from Pikud Haoref's public JavaScript files.

Unfortunately I have been stressed for time recently and cannot find the time to update it. Is this something you would like to take on yourself?

Thanks!

**Comment by eladnava** [comment]
Looking for contributors to complete this task, if anyone is interested please let me know. 👍

**Comment by DemonRem** [comment]
by the way @bennybauer 

if you know what wrong you can change you don't need elad to preform that

**Comment by Binternet** [comment]
@eladnava See if this helps
https://github.com/royts/israel-cities

**Comment by eladnava** [comment]
@Binternet Thanks so much for the suggestion, it could help but I think we should rely on an official source like Pikud Haoref for this kind of information rather than GitHub. Imagine there is some incorrect or outdated information in that JSON file, that would cause the alert mechanism fail.

**Comment by Binternet** [comment]
@eladnava This repo last commit is 5.5.2018, I created the pull request and took the data from https://data.gov.il/dataset/citiesandsettelments

**Comment by eladnava** [comment]
@Binternet Yep, and I think these are only available from Pikud Haoref's website:
`view-source:http://www.oref.org.il/894-en/Pakar.aspx`

Search for `var cities =`.

**Comment by eladnava** [comment]
Open-source `cities.json` scraper is pretty much complete ([cities.js](https://github.com/eladnava/pikud-haoref-api/blob/master/lib/cities.js))!

I have ran it a few times and verified most of the data, just need to verify further before overwriting the existing `cities.json`. :

**Comment by eladnava** [comment]
Updated and released in version `1.0.9.5`. 👍
